src/resources/postcss/views/list/_event.pcss

```diff
@@ -56,6 +56,7 @@
 
 		color: var(--color-text-secondary);
 		margin-bottom: calc(var(--spacer-0) * -1);
+		text-transform: uppercase;
```

Comment:
Do you feel it would be a good idea to add a mod class for uppercase and use it site wide?

Reply:
I'm not seeing a consistent enough pattern to say that it warrants a modifier class for uppercase. If you can point out a clear pattern, I wouldn't be opposed.

Reply:
I don't see a pattern either (I saw one for the month, day separators and the font weight), but I was just wondering if from a "framework" perspective it would be nice to have those.

Reply:
Worth asking the q, but sounds like based on the lack of a pattern, meh :)